src/unix/tty.c

```diff
@@ -120,6 +120,9 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, int fd, int readable) {
    * the handle queue, since it was added by uv__handle_init in uv_stream_init.
    */
 
+  if (!(flags & UV_STREAM_BLOCKING))
+    uv__nonblock(fd, 1);
+
 #if defined(__APPLE__)
```

Comment:
@saghul uv__nonblock is now above `#if defined(__APPLE__)` is intended?

This will affect all unix platforms right ?